contrib/logrotate/gitea.conf

```diff
@@ -0,0 +1,15 @@
+# Rotate Gitea logs
+# Replace path to logs if you use different working directory
+
+/var/lib/gitea/log/*.log, /var/lib/gitea/log/hooks/*.log {
+    daily
+    missingok
+    rotate 7
+    compress
+    notifempty
+    create 0660 git git
+    sharedscripts
+    postrotate
+        systemctl restart gitea.service
```

Comment:
This restart is unnecessary, it's probably enough to enable `copytruncate` and remove it.

Reply:
No no no,

Many application use restart-after-logrotate technique, and Gitea should too.

Reply:
That is a recipe for disaster. What if the application fails to restart?

Reply:
@silverwind hmm... `logrotate` should be blamed